### PR TITLE
Add tlds for th_TH internet provider

### PR DESCRIPTION
**Pull request by bact**
### What does this changes

- Add `tlds` for th_TH, in internet provider

### What was wrong

Internet provider previously not has th_TH locale

### How this fixes it

Add user name formats and TLDs for th_TH

### CODE REVIEW

**Review by fcurella — CHANGES_REQUESTED** [review]
Thank you! Do you mind adding a test for the new provider?

File: faker/providers/person/th_TH/__init__.py [hunk 1]
```diff
@@ -804,7 +804,7 @@ class Provider(PersonProvider):
         "ถนัดภาษา",
         "ถนัดรบ",
         "ถนัดรักษา",
-        "ถนัดหัดถกรรม",
+        "ถนัดหัตถกรรม",
```
Comment (fcurella) [inline faker/providers/person/th_TH/__init__.py:807]: I dont see how those two lines are different. Why are they sowing up in the diff?

Comment (bact) [reply faker/providers/person/th_TH/__init__.py:807]: They are two different characters, with a very similar shape (thus very common typo).
One is ด, another with ต.
The only difference is ต has a bend at the top curve of the character, but ด doesn't have the bend.

**Comment by bact** [comment]
I will add the test.

File: faker/providers/internet/th_TH/__init__.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            ("go.th", 40),
+            ("or.th", 40),
+            ("ac.th", 20),
+            ("net.th", 10),
+            ("mi.th", 5),
+            ("co", 10),
+            ("net", 20),
+            ("com", 150),
+            ("org", 50),
+        )
```
Comment (fcurella) [inline faker/providers/internet/th_TH/__init__.py:31]: can you add a trailing comma here?
```suggestion
        ),
```

**Review by fcurella — CHANGES_REQUESTED** [review]

**Review by fcurella — APPROVED** [review]
Thank you!

**Pull request merged**